### ISSUE TITLE
feature: Allow adjustment of paragraph spacing

### ISSUE THREAD

**Issue by jdittrich** [uxbox/uxbox#2428]
### Is your feature request related to a problem? Please describe.

When creating designs that deal with larger amounts of text, I want to be able to style the text. This includes adjusting the spacing between paragraphs

### Describe the solution you'd like.

Have an adjustment of paragraph spacing in the text section of the design panel

### Describe alternatives you've considered.

A very basic replacement is just adding line breaks

### Additional context

* Based on this forum post: https://community.penpot.app/t/better-text-styling-needed/1754/2
* When staying within SVG standards, this might be difficult. SVG 1.1. just assumes single line text; There might be [support for _line-height_ in SVG 1.2’s flow root](https://stackoverflow.com/questions/29829277/svg-how-to-set-text-line-height), but not for paragraph height.

**Comment by niwinz** [comment]
Yep, the SVG text support is a bit limiting. Penpot right now only uses the svg text support as implementation detail but the text initially is drawn using HTML (then with some dom magic we calculate the positions of the text and translate it to SVG text elements hehe).

I have opened an internal enhancement with this: https://tree.taiga.io/project/penpot/issue/4330
